ui/packages/consul-ui/app/components/consul/discovery-chain/utils.js

```diff
@@ -75,18 +76,18 @@ export const getResolvers = function(
     .forEach(function(item) {
       const parts = item.Name.split('.');
       let subset;
-      // this will leave behind the service.name.nspace.dc even if the service name contains a dot
+      // this will leave behind the service.name.nspace.partition.dc even if the service name contains a dot
       if (parts.length > 4) {
         subset = parts.shift();
       }
       parts.reverse();
-      // slice off from dc.nspace onwards leaving the potentially dot containing service name
+      // slice off from dc.partition.nspace onwards leaving the potentially dot containing service name
       // const nodeDc =
       parts.shift();
-      // const nodeNspace =
-      parts.shift();
       // const nodePartition =
       parts.shift();
+      // const nodeNspace =
+      parts.shift();
```

Comment:
Everything above here is just niceties, add partition into the comment etc The comments to explain/illustrate/document the shifting was actually out of order, but we don't use the resulting variables anyway so it didn't matter. Longer term we could probably use more destructuring here instead (this code was added when the codebase was in ES5 mode)